scripts/controlnet.py

```diff
@@ -894,29 +936,36 @@ def controlnet_main_entry(self, p):
                 model_net, control_model_type = Script.load_control_model(p, unet, unit.model)
                 model_net.reset()
 
+                if model_net is not None and getattr(devices, "fp8", False) and not isinstance(model_net, PlugableIPAdapter):
```

Comment:
This logic is applied to all existing ControlNet usages. Do we want to always enable fp8 when it is available?

Reply:
I’m not sure. But from my experiments I think fp8 is ok when CN+AD+A1111 are all in fp8